### ISSUE TITLE
Docker build error

### ISSUE THREAD

**Issue by geekscrapy** [target/strelka#71]
Bug occurs when building using: ```docker-compose -f build/docker-compose.yaml --project-name strelka up```

**Steps to reproduce**
1. Run ```docker-compose -f build/docker-compose.yaml --project-name strelka up```
2. Get error:
```
docker-compose -f build/docker-compose.yaml --project-name strelka up
Building frontend                                                                                                                                                                                              
Step 1/8 : FROM golang AS build                                                                                                                                                                                
 ---> 7ced090ee82e                                                                                                                                                                                             
Step 2/8 : LABEL maintainer "Target Brands, Inc. TTS-CFC-OpenSource@target.com"                                                                                                                                
 ---> Using cache                                                                                                                                                                                              
 ---> 7b9dc4e2e493                                                                                                                                                                                             
Step 3/8 : COPY ./src/go/ /go/src/github.com/target/strelka/src/go/                                                                                                                                            
 ---> Using cache                                                                                                                                                                                              
 ---> 6b281c63b673                                                                                                                                                                                             
Step 4/8 : RUN cd /go/src/github.com/target/strelka/src/go/cmd/strelka-frontend/ &&     go get . &&     CGO_ENABLED=0 go build -o /tmp/strelka-frontend .                                                      
 ---> Running in d8840028999d                                                                                                                                                                                  
# github.com/target/strelka/src/go/cmd/strelka-frontend                                                                                                                                                        
./main.go:82:24: cannot use redis.Z literal (type redis.Z) as type *redis.Z in argument to s.coordinator.client.cmdable.ZAdd                                                                                   
ERROR: Service 'frontend' failed to build: The command '/bin/sh -c cd /go/src/github.com/target/strelka/src/go/cmd/strelka-frontend/ &&     go get . &&     CGO_ENABLED=0 go build -o /tmp/strelka-frontend .' returned a non-zero code: 2
```

**Expected behavior**
Expected clean build in docker environment

**Release**
 - Commit ID in use: 3b31cd76ed0f06cdd75bc3bb90acfa1f26d2c419

**Additional context**
Host details:
```Linux mybuild 4.15.0-47-generic #50-Ubuntu SMP Wed Mar 13 10:44:52 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux```
```
Client:
 Version:           18.09.2
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        6247962
 Built:             Tue Feb 26 23:52:23 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.2
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.4
  Git commit:       6247962
  Built:            Wed Feb 13 00:24:14 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

**Comment by jshlbrd** [comment]
Thanks for reporting this, the go-redis package changed and is causing this error -- fix will come in soon.

**Comment by jshlbrd** [comment]
Fixed in #72
